VocaDbModel/VocaDb.Model.csproj

```diff
@@ -1,1240 +1,1265 @@
-﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Also this file has been completely regenerated for some reason. Can you fix it?

We should change the Model project to use SDK project format anyway (see #550), so it doesn't need to list all files. It would make changing project file less common.